var/spack/repos/builtin/packages/exawind/package.py

```diff
@@ -6,35 +6,118 @@
 from spack.package import *
 
 
-class Exawind(CMakePackage):
+class Exawind(CMakePackage, CudaPackage, ROCmPackage):
     """Multi-application driver for Exawind project."""
 
     homepage = "https://github.com/Exawind/exawind-driver"
     git = "https://github.com/Exawind/exawind-driver.git"
 
-    maintainers("jrood-nrel", "psakievich")
+    maintainers = ["jrood-nrel"]
 
     tags = ["ecp", "ecp-apps"]
 
-    version("master", branch="main")
+    # Testing is currently always enabled, but should be optional in the future
+    # to avoid cloning the mesh submodule
+    version("master", branch="main", submodules=True)
 
-    depends_on("trilinos+stk")
-    depends_on("tioga+shared~nodegid")
-    depends_on("nalu-wind+hypre+openfast+tioga+wind-utils")
-    depends_on("amr-wind+hypre+mpi+netcdf+openfast")
-    depends_on("openfast+cxx+shared@2.6.0:")
+    variant("openfast", default=False, description="Enable OpenFAST integration")
+    variant("hypre", default=True, description="Enable hypre solver")
+    variant("stk_simd", default=False, description="Enable SIMD in STK")
+    variant("umpire", default=False, description="Enable Umpire")
+    variant("tiny_profile", default=False, description="Turn on AMR-wind with tiny profile")
+    variant("sycl", default=False, description="Enable SYCL backend for AMR-Wind")
+    variant("gpu-aware-mpi", default=False, description="gpu-aware-mpi")
+
+    conflicts("+hypre", when="+sycl")
```

Comment:
hypre can build with `+sycl` so I think we can remove this.

Reply:
I configured this so that `+sycl` is `amr-wind~hypre+sycl` and `nalu-wind+hypre`  with nalu-wind on host.

Reply:
This did need to go though. I added some more sycl clean up.

Reply:
> hypre can build with +sycl so I think we can remove this.

Last I tried - `hypre+sycl` didn't work for me.

https://github.com/spack/spack/pull/40722#issuecomment-1792966471

Reply:
It may have very well gotten broken, and I haven't tried the latest Spack, but I use it successfully on the Aurora machine.